### PR TITLE
feat(docs): detailed, shareable and downloadable docs

### PR DESCRIPTION
**Pull request by ilyozzz**

### CODE REVIEW

File: services/actions/src/rpc/genSchemasDocs.js [hunk 1]
```diff
@@ -53,53 +54,88 @@ const setMarkdownDocMutation = `
...
+  }
+
+  return `${text(title, { bold: true, indent: 2, postNewLine: false })}: \`${source[key]}\``;
+};
+
+const generateMemberDoc = member => {
+  const { meta } = member;
+
+  let result = header(member.shortTitle, { size: 5, bold: true, indent: 8 });
+  result += text(member.description || 'No description provided', { indent: 8 });
```
Comment (ifokeev) [inline services/actions/src/rpc/genSchemasDocs.js:81]: there could be `description` for the cube itself too, and we have to support this

File: services/actions/src/rpc/genSchemasDocs.js [hunk 2]
```diff
@@ -53,53 +54,88 @@ const setMarkdownDocMutation = `
...
+  result += text('Parameters:', { bold: true, indent: 8 });
+
+  const parameters = [
```
Comment (ifokeev) [inline services/actions/src/rpc/genSchemasDocs.js:84]: we need to show the column name for dimensions

File: services/client/src/pages/Docs.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  const { versionId } = params;
+  
+  if (!versionId) {
+    return <ErrorFound status={404} />;
+  }
+
+  return (
+    <Container>
+      <div style={{ padding: 40 }}>
+        <DocsTab versionId={versionId} />
```
Comment (ifokeev) [inline services/client/src/pages/Docs.js:19]: could we show "download markdown" on this page?

Comment (ifokeev) [reply services/client/src/pages/Docs.js:19]: Could we have all scaffolds open by default?

File: services/client/src/components/DocsTab.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    downloadBlob(blob, `${versionId}.md`);
+  }, [markdown, versionId]);
+
+  if (docData.loading) {
+    return <Loader spinning />;
+  }
+
+  return (
+    <>
+      { toolbar && (
```
Comment (ifokeev) [inline services/client/src/components/DocsTab.js:33]: let's place `(docs)` link near the branch title, keep docs as a tab it not helpful, in my opinion

**Review by ifokeev — CHANGES_REQUESTED** [review]

File: services/actions/src/rpc/genSchemasDocs.js (hunk 2 above)
Comment (ilyozzz) [reply services/actions/src/rpc/genSchemasDocs.js:84]: there's `sql` field in the params. Is that what you mean?

Comment (ifokeev) [reply services/actions/src/rpc/genSchemasDocs.js:84]: yes

Comment (ifokeev) [reply services/actions/src/rpc/genSchemasDocs.js:84]: still not showing `sql` data

**Review by ifokeev — APPROVED** [review]

**Pull request merged**